provision/000_base/roles.yaml

```diff
@@ -5,3 +5,4 @@ roles:
   # placeholder, admin role is created programmatically
   # with a fixed ID before configs are processed
   - admins
+  - federation
```

Comment:
Shouldn't we also define the ID inside `pkg/rbac/permissions.go`?
Also add it here `pkg/provision/roles.go`